index.html

```diff
@@ -338,9 +339,47 @@ <h5 class="language">Skills</h5>
                               </div>
                         </div>
                     </div> 
-                </section>
-          </footer> 
+     
+                <!-- </section> -->
+                <!-- ---------------------------------------------------------------End----------------------------------------------------------------- -->
+                
+      </section> 
+           <!-- -------------------------------------------------------------contact form ---------------------------------------------------------- -->
+           <section class="contact-section ">
+            <div class="contact-content">
+                    <div class="contact-me">
+                      <h2>contact me</h2>
+                    </div>
+                    <div class="about-us">
+                          <p>
+                            If you have an application you are interested in developing, 
+                            a feature that you need built or a project that needs coding. 
+                            I’d love to help with it!
+                          </p>
+                    </div>
+              
+                  <form action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] Great job 👏 adding the formspree link but it seems like you've used the boilerplate link here the "form_id" should be the ID you receive after signing up on [formspree](https://formspree.io/html) so please fix that ✔.